tutorials/combine-feature-collections/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Combine FeatureCollections into a new FeatureCollection
+description: This basic tutorial shows how users can combine two FeatureCollections into one. It is targeted at beginners.
+author: sabrinaszeto
+tags: beginner, featurecollection
+date_published: 2019-09-18
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/b4a2877c24cdd175bca2d7252cd733de)
+
+This basic tutorial shows how users can combine two `ee.FeatureCollection`s into a new `ee.FeatureCollection`.
+
+## Create two `ee.FeatureCollection` objects
+
+Let's begin by generating two sets of random points within the boundary of Utah state in USA. First, define the boundary of Utah as a geometry.
+
+```
+var UtahGEOMETRY = ee.Geometry.Polygon([
+    [-114.05, 37],
+    [-109.05, 37],
+    [-109.05, 41],
+    [-111.05, 41],
+    [-111.05, 42],
+    [-114.05, 42]
+]);
+```
+
+Then, generate two sets of different random points containing 25 points each. We ensure that the points are different by using a different seed, namely 12 and 1, to generate each set.
+```
+var NewFEATURES = ee.FeatureCollection.randomPoints(UtahGEOMETRY, 25, 12);
```

Comment:
Please use lowerCamelcase for all variable names (as opposed to MixedCASE or ALLCAPS)

Reply:
Will work on this!